boms/cloud-oss-bom/RELEASING.md

```diff
@@ -32,6 +32,8 @@ version in these documents:
 * https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM
 * https://github.com/googleapis/google-http-java-client/blob/master/docs/setup.md
 * https://github.com/googleapis/google-cloud-java/blob/master/TROUBLESHOOTING.md
+* Merge the dependabot PR that updates libraries-bom in https://github.com/GoogleCloudPlatform/java-docs-samples/pulls
+* Manually edit and update any pom.xml files in https://github.com/GoogleCloudPlatform/java-docs-samples that dependabot missed
```

Comment:
These two bullet points list actions. The first three are URL of the document. Would remove "Merge ..." from the list?

```suggestion

On the next day, merge the dependabot PR that updates libraries-bom in https://github.com/GoogleCloudPlatform/java-docs-samples/pulls.
Then manually edit and update any pom.xml files in https://github.com/GoogleCloudPlatform/java-docs-samples that dependabot missed
```

Reply:
Can you add `find . -name pom.xml |xargs grep -A 1 '<artifactId>libraries-bom'`?

Reply:
No, that's not how I did it or would want to do it.

Reply:
How do you want to do it?

Reply:
I used Github search but that's a choice for whoever's releasing. find works too if you prefer that. No need to specify it here.